### DISCUSSION TITLE
2 issues in 'translator mode', any trick?

### DISCUSSION THREAD

**Discussion by PreetM07** [SubtitleEdit/subtitleedit#6482] [Q&A]
So, I found 2 issues while working with original translator's mode...
1. Undo doesn't work.
2. Video preview doesn't keep the font size for this mode and goes to the default and changing it doesn't have any effect.

**Comment by PreetM07** [comment]
3. Toggle focus, list view and subtitle text box... doesn't work.

**Comment by darnn** [comment]
I'm not sure what you mean by 2, but as for 1, there are indeed some issues with the undo/redo function. But they only show up under fairly specific circumstances, which I haven't been able to pinpoint well enough to describe in a bug report. The quick and easy (though obviously imperfect) solution is to restart SE, as it will take a while for the problem to surface again.
And could you expand a bit on 3? How are you trying to toggle focus and how exactly does it not work? And are you using the most recent version?
